### DISCUSSION TITLE
Is this normal? Or is it a mistake?

### DISCUSSION THREAD

**Discussion by Nothnq** [IBAX-io/go-ibax#1141] [Bug Submission]
![ibax-111](https://user-images.githubusercontent.com/98263284/151413667-5f7c6c39-fffe-4fa9-ad52-fdd4bd0ec306.jpg)

![ibax-222](https://user-images.githubusercontent.com/98263284/151413685-250c9606-11b5-486a-8763-2dd1091c82a0.jpg)

**Comment by yasinogur** [comment]
why are there two items? 

there are one item on the my wallet that part of  Member information

i didnt get

**Comment by ibaxio** [comment]
please show the full screen.

Did you join an ecosystem?
